learn/cockpit-learn.yaml

```diff
@@ -46,3 +47,14 @@ items:
       port: 80
       targetPort: 8080
       protocol: TCP
+- kind: Route
+  apiVersion: v1
+  metadata:
+    name: cockpit-learn
+  spec:
+    selector:
+      infra: cockpit-learn
+    path: "/predict"
```

Comment:
Is that intentional? prediction and learning are two opposite functions of this, having it in the same pod feels weird? Or do we run learning (log scanning and json creation) and prediction (REST API to map a log to a "flake?") on the same container?